modules/microshift-install-bootc-build-image.adoc

```diff
@@ -14,8 +14,9 @@ Image mode for {op-system-base} is Technology Preview. Using a bootc image in pr
 ====
 
 .Prerequisites
-* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
-* You are logged into the {op-system-base} 9.4 host using the user credentials that have `sudo` permissions.
+
+* A {op-system-base-full} {op-system-version} host with an active Red{nbsp}Hat subscription for building {microshift-short} bootc images and running containers.
```

Comment:
This is rendered to 9.6 version, which is wrong in 4.18. Should be 9.4.

Reply:
Yes, the main branch is updated for 4.19 already. We can verify after merge that it renders correctly in 4.18.